doc/configuration/outgoing-connections.md

```diff
@@ -435,7 +435,14 @@ Cipher suites to use. For allowed values, see the [Erlang/OTP SSL documentation]
 
 ### `outgoing_pools.*.*.connection.tls.server_name_indication`
 * **Syntax:** boolean
-* **Default:** `true`
+* **Default:** `false`, but enabled if the `verify_peer` option is set to `true`
 * **Example:** `tls.server_name_indication = false`
 
-Enables SNI extension to TLS protocol.
+Enables SNI extension to TLS protocol. If set to `true`, the `server_name_indication_host` option should be provided.
+
+### `outgoing_pools.*.*.connection.tls.server_name_indication_host`
+* **Syntax:** string
+* **Default:** not set
+* **Example:** `tls.server_name_indication_host = "domain.com"`
+
+Domain against which the certificates will be checked, using SNI.
```

Comment:
```suggestion
Domain against which the certificates will be checked, using SNI. It can be specified only when `server_name_indication` is set to `true`.
```